Command/PasswordReset.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Command;
+
+use SumoCoders\FrameworkMultiUserBundle\User\UserInterface;
+
+class PasswordReset
```

Comment:
Een command is een actie die gebeurd door een user. `ResetPassword` lijkt me dan een logischer naam.